wdtk-wikibaseapi/src/test/java/org/wikidata/wdtk/wikibaseapi/TermStatementUpdateTest.java

```diff
@@ -119,6 +121,7 @@ public void testAddLabelAndAlias() throws JsonProcessingException {
 		assertEquals(Collections.singleton("fr"), su.getAliasUpdates().keySet());
 		assertEquals(alias.getText(), su.getAliasUpdates().get("fr").get(0).getText());
 		assertTrue(su.getDescriptionUpdates().isEmpty());
+		assertTrue(!su.isNull());
```

Comment:
assertFalse would be more readable.

Reply:
for some reason Eclipse was telling me it didn't exist (not sure why though)